docs/next.config.js

```diff
@@ -158,6 +158,7 @@ module.exports = withDocsInfra({
     SLACK_FEEDBACKS_TOKEN: process.env.SLACK_FEEDBACKS_TOKEN,
     SOURCE_CODE_ROOT_URL: 'https://github.com/mui/material-ui/blob/master', // #default-branch-switch
     SOURCE_CODE_REPO: 'https://github.com/mui/material-ui',
+    GITHUB_TEMPLATE_DOCS_FEEDBACK: '4.docs-feedback.yml',
```

Comment:
I'm adding this variable because github template will not necessarily have the same name on each repo. For example on X we already have a 4.xxxx.yml template